.github/workflows/build.yml

```diff
@@ -825,20 +773,10 @@ jobs:
         run: |
           docker run --name=front-test --net=host -v $PWD/output:/output \
             ${{ fromJSON(needs.build.outputs.stable_tags).front-tests }} \
-            sh -c "npm run test-coverage && cp -R coverage /output"
```

Comment:
We could get rid of the `test-coverage` script in `package.json`. Potentially drop the `@vitest/coverage-v8` dependency?